Makefile

```diff
@@ -120,51 +142,68 @@ DEPENDS	:=	$(OFILES:.o=.d)
 #---------------------------------------------------------------------------------
 # main targets
 #---------------------------------------------------------------------------------
-all: $(OUTPUT).gcb $(OUTPUT).vgc $(OUTPUT).gci $(OUTPUT)_xz.gci
 
+#---------------------------------------------------------------------------------
+# Base DOL (PicoBoot)
+#---------------------------------------------------------------------------------
 $(OUTPUT).dol: $(OUTPUT).elf
 $(OUTPUT).elf: $(OFILES)
 
 $(OFILES_SOURCES) : $(HFILES)
 
-%.gcb: %.dol
-	@echo pack IPL ... $(notdir $@)
-	@cd $(PWD); ./dol2ipl.py ipl.rom $< $@
-
+#---------------------------------------------------------------------------------
+# GCI
+#---------------------------------------------------------------------------------
 %.gci: %.dol
-	@echo dol2gci ... $(notdir $@)
+	@echo converting DOL to GCI ... $(notdir $@)
 	@dol2gci $< $@ boot.dol
 
-$(OUTPUT)_xz.dol: $(OUTPUT).dol
-	@echo compress ... $(notdir $@)
-	@dolxz $< $@ -cube
-
-$(OUTPUT)_xz.elf: $(OUTPUT)_xz.dol
-	@echo dol2elf ... $(notdir $@)
-	@doltool -e $<
+#---------------------------------------------------------------------------------
+# Qoob
+#---------------------------------------------------------------------------------
+$(OUTPUT).gcb: $(OUTPUT).dol
+	@echo packing Qoob IPL ... $(notdir $@)
+	@cd $(PWD); ./dol2ipl.py ipl.rom $< $@
 
-%.qbsx: %.elf
-	@echo pack IPL ... $(notdir $@)
+#---------------------------------------------------------------------------------
+# Qoob SX
+#---------------------------------------------------------------------------------
+$(OUTPUT)_xz.qbsx: $(OUTPUT)_xz.elf
+	@echo packing Qoob SX IPL ... $(notdir $@)
 	@cd $(PWD); ./dol2ipl.py /dev/null $< $@
 
 $(OUTPUT_SX).elf: $(OUTPUT)_xz.qbsx
-	@echo splice ... $@
+	@echo splicing Qoob SX bios ... $(notdir $@)
 	@cd $(PWD); cp -f qoob_sx_13c_upgrade.elf $@
 	@cd $(PWD); dd if=$< of=$@ obs=4 seek=1851 conv=notrunc
 	@cd $(PWD); printf 'QOOB SX iplboot install\0' \
 		| dd of=$@ obs=4 seek=1810 conv=notrunc
 
-%.vgc: %.dol
-	@echo pack IPL ... $(notdir $@)
+#---------------------------------------------------------------------------------
+# Viper
+#---------------------------------------------------------------------------------
+$(OUTPUT).vgc: $(OUTPUT).dol
+	@echo packing Viper IPL... $(notdir $@)
 	@cd $(PWD); ./dol2ipl.py /dev/null $< $@
 
-%_xeno.elf: $(OFILES)
-	@echo linking ... $(notdir $@)
+#---------------------------------------------------------------------------------
+# Xeno
+#---------------------------------------------------------------------------------
+$(OUTPUT)_xeno.dol: $(OUTPUT)_xeno.elf
+$(OUTPUT)_xeno.elf: $(OFILES)
+	@echo linking Xeno ... $(notdir $@)
 	@$(LD)  $^ $(LDFLAGS) -Wl,--section-start,.init=0x81700000 $(LIBPATHS) $(LIBS) -o $@
 
-%.bin: %.elf
-	@echo extract binary ... $(notdir $@)
-	@$(OBJCOPY) -O binary $< $@
```

Comment:
I don't see a replacement for this, but I'm pretty sure it was for Xeno so it's dead code anyway.

Reply:
Yea it was not used or referenced anywhere so I assumed it was dead and removed it.